core/src/main/resources/lib/layout/task.jelly

```diff
@@ -77,6 +77,17 @@ THE SOFTWARE.
       (onclick supersedes this except in the context menu.)
       (since 1.512)
     </st:attribute>
+    <st:attribute name="onclick" deprecated="true">
+      Onclick inline JS handler. Deprecated, specify data-callback attribute instead.
+    </st:attribute>
+    <st:attribute name="data-callback" since="TODO">
+      Name of a global function to call when clicked.
+      It will be called with an element (currently 'a') as first argument, and the event as second argument.
+      You can specify further data-* attributes to customize behavior, those will be defined on the element passed as first argument.
```

Comment:
I had to look up https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset so it would be nice to make that explicit.